README.md

```diff
@@ -0,0 +1,406 @@
+[![Supported Python versions](https://img.shields.io/pypi/pyversions/pypinfo.svg?style=flat-square)](https://pypi.org/project/pypinfo)
+[![License](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://en.wikipedia.org/wiki/MIT_License)
+[![Code style: Black](https://img.shields.io/badge/code%20style-black-000000.svg?style=flat-square)](https://github.com/psf/black)
+
+pypinfo is a simple CLI to access [PyPI](https://pypi.org/) download statistics via Google's BigQuery.
+
+## Table of contents
+
+1. [Usage](#usage)
+2. [How to run](#installation)
+3. [Credits](#credits)
+
+## Usage
+
+<details>
+  <summary>
+  Click to unfold usage
+  </summary>
+
+```console
+$ pypinfo
+    Usage: pypinfo [OPTIONS] [PROJECT] [FIELDS]... COMMAND [ARGS]...
+
+        Valid fields are:
+
+        project | version | file | pyversion | percent3 | percent2 | impl | impl-version |
+
+        openssl | date | month | year | country | installer | installer-version |
+
+        setuptools-version | system | system-release | distro | distro-version | cpu |
+
+        libc | libc-version
+
+    Options:
+        -a, --auth TEXT         Path to Google credentials JSON file.
+        --run / --test          --test simply prints the query.
+        -j, --json              Print data as JSON, with keys `rows` and `query`.
+        -i, --indent INTEGER    JSON indentation level.
+        -t, --timeout INTEGER   Milliseconds. Default: 120000 (2 minutes)
+        -l, --limit TEXT        Maximum number of query results. Default: 10
+        -d, --days TEXT         Number of days in the past to include. Default: 30
+        -sd, --start-date TEXT  Must be negative or YYYY-MM[-DD]. Default: -31
+        -ed, --end-date TEXT    Must be negative or YYYY-MM[-DD]. Default: -1
+        -m, --month TEXT        Shortcut for -sd & -ed for a single YYYY-MM month.
+        -w, --where TEXT        WHERE conditional. Default: file.project = "project"
+        -o, --order TEXT        Field to order by. Default: download_count
+        --all                   Show downloads by all installers, not only pip.
+        -pc, --percent          Print percentages.
+        -md, --markdown         Output as Markdown.
+        -v, --verbose           Print debug messages to stderr.
+        --version               Show the version and exit.
+        -h, --help              Show this message and exit.
+```
+
+pypinfo accepts 0 or more options, followed by exactly 1 project, followed by
+0 or more fields. By default only the last 30 days are queried. Let's take a
+look at some examples!
+
+_Tip_: If queries are resulting in NoneType errors, increase timeout.
+
+### Downloads for a project
+
+```console
+    $ pypinfo requests
+    Served from cache: False
+    Data processed: 2.83 GiB
+    Data billed: 2.83 GiB
+    Estimated cost: $0.02
+
+    | download_count |
+    | -------------- |
+    |    116,353,535 |
```

Comment:
Dedent all these like this?

```suggestion
$ pypinfo requests
Served from cache: False
Data processed: 2.83 GiB
Data billed: 2.83 GiB
Estimated cost: $0.02

| download_count |
| -------------- |
|    116,353,535 |
```

Otherwise it looks like:

<img width="307" alt="image" src="https://user-images.githubusercontent.com/1324225/224503620-a72dfc59-e6a8-4b6a-b20b-a47b3f387cc0.png">